radixdlt/src/main/java/com/radixdlt/consensus/ValidatingEventCoordinator.java

```diff
@@ -217,17 +218,21 @@ public void processProposal(Proposal proposal) {
 		}
 
 		final View updatedView = this.pacemaker.getCurrentView();
-		if (proposedVertex.getView().compareTo(updatedView) != 0) {
-			log.info("{}: PROPOSAL: Ignoring view {} Current is: {}", this.getShortName(), proposedVertex.getView(), updatedView);
+		if (proposedVertexView.compareTo(updatedView) != 0) {
+			log.info("{}: PROPOSAL: Ignoring view {} Current is: {}", this.getShortName(), proposedVertexView, updatedView);
 			return;
 		}
 
+		if (proposedVertex.getParentView() != null && proposedVertexView.number() != proposedVertex.getParentView().number() + 1) {
```

Comment:
proposedVertex.getParentView() should never be null. This should be a validation/assertion check instead if it is to be used.

Also, Vertex has `.hasDirectParent()` which can be used instead